src/_gettsim/transfers/rente.py

```diff
@@ -288,7 +284,7 @@ def ges_rente_zugangsfaktor(  # noqa: PLR0913
 
     """
 
-    if rentner and ges_rente_vorauss_regelrente:
```

Comment:
Because `ges_rente_m` returns non-zero pensions only if `rentner` is `True`, I think it is not necessary to set the Zugangsfaktor to 0 as well.